FirebaseAuth/Sources/Swift/User/User.swift

```diff
@@ -2102,8 +2113,8 @@ extension User: NSSecureCoding {}
             of: NSString.self,
             forKey: kFirebaseAppIDCodingKey
           ) as? String,
-          let tokenService = coder.decodeObject(forKey: kTokenServiceCodingKey)
-          as? SecureTokenService else {
+          let tokenService = coder.decodeObject(of: SecureTokenService.self,
+                                                forKey: kTokenServiceCodingKey) else {
```

Comment:
Decoding from the keychain returns nil without the `of`. Unfortunately, it's not needed in the unit tests.

Reply:
My understanding is the `SecureTokenService` provides api to read from the keychain. Do you know why instances of the class are decoded, rather than initialized when Auth is created?

Reply:
The service stores tokens in properties and it looks like those should be decoded from the keychain to avoid regenerating.